src/tui.rs

```diff
@@ -161,6 +161,8 @@ use crate::{apperr, arena_format, input, unicode};
 
 const ROOT_ID: u64 = 0x14057B7EF767814F; // Knuth's MMIX constant
 const SHIFT_TAB: InputKey = vk::TAB.with_modifiers(kbmod::SHIFT);
+const KBMOD_FOR_WORD_NAV: InputKeyMod =
+    if cfg!(target_os = "macos") { kbmod::ALT } else { kbmod::CTRL };
```

Comment:
Made the name a bit more specific so it's easier to remember what it stands for in future reviews.